os/drivers/testcase/test_kthread_stack_protection.c

```diff
@@ -0,0 +1,171 @@
+/****************************************************************************
```

Comment:
Please rebase the code base. This file should be located in `os/drivers/kernel_test`

Reply:
done